apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/Inscriptions/DetailsDrawer/Actions.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { t } from "i18next";
```

Comment:
Get it from your component like this instead : `const { t } = useTranslation();`

Reply:
✅